extension/storage/go.mod

```diff
@@ -3,7 +3,7 @@ module github.com/open-telemetry/opentelemetry-collector-contrib/extension/stora
 go 1.20
 
 require (
-	github.com/jackc/pgx/v4 v4.18.1
+	github.com/jackc/pgx/v4 v4.12.1-0.20210724153913-640aa07df17c
```

Comment:
This PR states that it is bumping to v5 but it's not. The CI is also failing with

```
Generated code is out of date, please run "make genotelcontribcol" and commit the changes in this PR.
```

We should investigate before merging